ports/esp8266/boards/GENERIC_512K/manifest.py

```diff
@@ -1,5 +1,4 @@
 freeze("$(BOARD_DIR)", "_boot.py", opt=3)
-freeze("$(PORT_DIR)/modules", ("apa102.py", "ntptime.py", "port_diag.py"))
```

Comment:
this looks like it removes things that don't have anything to do with this PR?

Reply:
I needed to remove `ntptime.py` from this list, then spotted that the manifest in the parent directory was already freezing everything in `$(PORT_DIR)/modules`, so this entire line was already redundant. At that point, it seemed a bit silly to just remove `ntptime.py` rather than the whole line.

Reply:
OK, sounds good.

Reply:
@daviesian @dpgeorge This file explicitly overrides the one in the parent directory (it doesn't `include()` the parent one, like for example `GENERIC/manifest.py` does).

The idea being that the 512k "board" includes fewer modules by default, so it doesn't automatically get the contents of the modules dir.

Reply:
Added a fix in #9037